packages/create-redwood-app/template/web/tsconfig.json

```
@@ -31,6 +31,6 @@
     "src",
     "../.redwood/types/includes/all-*",
     "../.redwood/types/includes/web-*",
-    "../types"
+    "./types"
```

```suggestion
    "../types",
    "./types"
```

If you look at the `types/*` entry under `paths` above you'll notice that it lists both `./types` and `../types`. So I think we should have both here as well. What do you think?

---

That's cool, done!